config/resourcefulbees/bees/bees_gem/bred/Fluix.json

```diff
@@ -0,0 +1,36 @@
+{
+    "flower": "appliedenergistics2:fluix_block",
+    "maxTimeInHive": 700,
+    "hasHoneycomb": true,
+    "ColorData": {
+        "isBeeColored": true,
+        "honeycombColor": "#411985",
+        "primaryColor": "#411985",
+        "secondaryColor": "#4f1985"
+    },
+    "MutationData": {
+        "hasMutation": false
+    },
+    "CentrifugeData": {
+        "hasCentrifugeOutput": true,
+        "mainOutput": "appliedenergistics2:purified_fluix_crystal",
+        "mainOutputCount": 1,
+        "mainOutputWeight": 0.15,
+        "secondaryOutput": "appliedenergistics2:purified_fluix_crystal",
+        "secondaryOutputWeight": 0.01,
+        "secondaryOutputCount": 1
+    },
+    "SpawnData": {
+        "canSpawnInWorld": false
+    },
+    "BreedData": {
+        "isBreedable": true,
+        "parent1": "Certus",
+        "parent2": "Redstone",
+        "feedItem": "appliedenergistics2:purified_fluix_crystal",
+        "feedAmount": 4
+    },
+    "TraitData": {
+        "hasTraits": false
+    }
+}
```

Comment:
We discussed briefly in dev chat, and the few of us present agreed that we do not want a bee for a composite type like Fluix. We defer to @WhitePhant0m though.

Reply:
Asked Phant0m, he said nay.

Reply:
Yeah, I don't think a fluix bee would be good, and I will probably be nuking the fluix plant too.

Reply:
Fair enough, made the fluix bee since we had the fluix crop.